pom.xml

```diff
@@ -91,6 +91,12 @@
 	<repositories>
 
 		<repository>
+			<id>tychoM2E</id>
+			<layout>p2</layout>
+			<url>http://download.jboss.org/jbosstools/updates/requirements/m2eclipse-tycho/0.8.1.201704211436/</url>
```

Comment:
That is a test. I am looking for a temp. workaround. The m2e stuff will move into the JBT TP anyways soon and therefore out of the JBTIS TP. Once that is done I will remove the definition....or alternatively if the addition did not help in my test :D